### DISCUSSION TITLE
text substitution

### DISCUSSION THREAD

**Discussion by pwm1234-sri** [dotnet/docfx#8262] [General]
**Operating System**: (`Windows` )

**DocFX Version Used**:  2.59.2.0

**Template used**: `statictoc`

**Steps to Reproduce**:

I have the same content that I need in multiple places where ``inheritdoc`` does not work (or at least I do not know how to use it). 
For example, I have a description of a function that I want in both the xml documentation comment and in a conceptual markdown document. How do I get the same text in a conceptual markdown document and in an xml documentation comment? For example, I want to have the content of the ``<summary>`` doc comment in a conceptual markdown document that is providing the overview of a method. With docfx, is there a way to get the exact same content in both an xml doc comment (like ``<summary>``) into a markdown conceptual document.

I am familiar with both sphinx/rst and doxygen. They each have a way to do this. sphinx has [substitutions](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions) and doxygen has [aliases](https://www.doxygen.nl/manual/custcmd.html#custcmd_simple).
Is there something similar for docfx with C#?

**Comment by KalleOlaviNiemitalo** [comment]
DocFX has a [file inclusion](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#file-inclusion) feature in Markdown but I'm not sure whether you can use that within XML documentation comments; does DocFX process Markdown in those elements when it generates HTML, and would anything expand it in the XML file that you publish for Intellisense? Perhaps the `summary` would have to use an [`include`](https://docs.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags#include) XML element instead.

**Comment by pwm1234-sri** [comment]
Thanks for your response, but I do not think that the DFM file inclusion and the xml comment ``<include>`` tag can share content. If they can, can someone provide an example? (Also, if they could, I think that would mean each phrase that needs to be shared would have to be in its own file - which would make it possible, but I do not understand those two features as being compatible.)
